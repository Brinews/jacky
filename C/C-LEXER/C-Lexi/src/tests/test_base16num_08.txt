  000000000123FAx16
