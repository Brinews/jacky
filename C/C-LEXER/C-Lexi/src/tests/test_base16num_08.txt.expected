{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{0 , NUM , 1}
{123FAx16 , BASE16NUM , 1}
{ , END_OF_FILE , 3}
